Joomla/Sniffs/Commenting/ClassCommentSniff.php

```diff
@@ -6,18 +6,17 @@
  * @copyright  Copyright (C) 2015 Open Source Matters, Inc. All rights reserved.
  * @license    http://www.gnu.org/licenses/gpl-2.0.txt GNU General Public License Version 2 or Later
  */
+namespace Joomla\Sniffs\Commenting;
 
-if (class_exists('Joomla_Sniffs_Commenting_FileCommentSniff', true) === false)
-{
-	throw new PHP_CodeSniffer_Exception('Class Joomla_Sniffs_Commenting_FileCommentSniff not found');
-}
-
+use PHP_CodeSniffer\Sniffs\Sniff;
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Util\Tokens;
 /**
```

Comment:
I'm not sure, but based on [the error in travis](https://travis-ci.org/joomla/coding-standards/jobs/254911561#L492) I think you need to include a `use Joomla\Sniffs\Commenting\FileCommentSniff` since we are extending from `FileCommentSniff`.